lib/hubspot/config.rb

```diff
@@ -47,6 +48,11 @@ def ensure!(*params)
           raise Hubspot::ConfigurationError.new("'#{p}' not configured") unless instance_variable_get "@#{p}"
         end
       end
+
+      private
```

Comment:
How would you feel about adding a newline between this `private` line and the next method definition?

Reply:
Done 👍